front/src/modules/ui/dropdown/components/DropdownMenu.tsx

```diff
@@ -16,6 +16,8 @@ export const DropdownMenu = styled.div<{
 
   flex-direction: column;
 
+  max-width: 160px;
```

Comment:
this is not needed anymore as we are providing width on line 23

Reply:
Have removed it.